src/components/Profile/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ProfileContextProvider } from './Context'
+import { Name } from './Name'
+import { SocialNetwork } from './SocialNetwork'
+import { Email } from './Email'
+import { ProfileButton, Center, Space, Wrapper } from './styled'
+import { AboutMe } from './AboutMe'
+
+type Props = {
+	fullName: string
+	email: string
+}
+
+const ContentProfile: React.FC<Props> = ({ fullName, email }) => (
+	<ProfileContextProvider addresses={[]}>
+		<Wrapper>
+			<Center>
+				<Name fullName={fullName} />
+				<Email email={email} />
+				<Space />
+				<AboutMe />
+				<Space />
+				<SocialNetwork />
+				<Space />
+				<ProfileButton titleSmall="Přidat do kontaktů" />
+				<Space />
+				<ProfileButton title="Odhlásit se" />
```

Comment:
Buttons are centred on mobile but on the left side of the page on desktop. Is this planned or a bug?

Reply:
Intention. Center alignment looks better on a small display.